readme-cmake.md

```diff
@@ -80,7 +80,7 @@ If you want to build the libraries from source instead of using Homebrew, you ca
 
 ## Linux <Linux>
 
-It is recommended to use a newish CMake, although CMake 3.16 is the lowest supported. You can get the latest version from the [offical website](https://cmake.org/download/). To follow the instructions written below, you will need at least CMake 3.21.
+You will need at least CMake 3.21. You can get the latest version from the [offical website](https://cmake.org/download/).
```

Comment:
```suggestion
You will need at least CMake 3.21. You can get the latest version from the [offical website](https://cmake.org/download/).

> **Please note:** if your distro contains an out of date CMake you can download an updated version directly and it will still likely work on your distribution.
```

Reply:
Nevermind; irrelevant feedback